### PR TITLE
IO register viewer

### PR DESCRIPTION
**Pull request by Superfranci99**
Engine A hasn't been finished yet (need to work with bits too)

Implements partially #33

### CODE REVIEW

File: IO_RegisterViewer/EngineA.cs [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+	public class EngineA : DataStructure
+	{
+		public override void Write ()
+		{
+			throw new NotImplementedException ();
+		}
+
+		public override void Read ()
+		{
+			GdbStream stream = new GdbStream (GdbSessionManager.GetClient(0));
```
Comment (pleonex) [inline IO_RegisterViewer/EngineA.cs]: Should use this:
``` c#
GdbStream stream = GdbSessionManager.GetDefaultClient().Stream;
```

File: IO_RegisterViewer/EngineA.cs [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+		public short    Dispstat       { get; set; }
+		public short    Vcount         { get; set; }
+		public Engine2D Engine2dA      { get; set; }
+		// 
+		public short    Disp3dcnt      { get; set; }
+		public int      Dispcapcnt     { get; set; }
+		public int      Disp_mmem_fifo { get; set; }
+		public short    Master_bright  { get; set; }
+
+		#region bitter
```
Comment (pleonex) [inline IO_RegisterViewer/EngineA.cs]: Empty region? XD

Comment (Superfranci99) [inline IO_RegisterViewer/EngineA.cs]: ops I've forgotten to remove it xD
it was a blank line to remember me that I was missing 4 bytes somewhere xD

**Comment by pleonex** [comment]
Also, remove *bin* and *obj* folder. I will add some lines in *.gitignore* to ommit these directories.

**Pull request merged**